src/wasm-lib/kcl/tests/artifact_graph_sketch_on_face_etc/artifact_graph_mind_map.snap.md

```diff
@@ -221,15 +41,12 @@ mindmap
           Wall
           SweepEdge Opposite
           SweepEdge Adjacent
-        Segment
-          Wall
-          SweepEdge Opposite
-          SweepEdge Adjacent
         Segment
         Sweep Extrusion
           Wall
           Wall
           Wall
+          Cap Start
```

Comment:
I don't think this is a real addition. I think it's misleading because of how the mind map is created.